### ISSUE TITLE
Create demo page with component usage 

### ISSUE THREAD

**Issue by GoNZooo** [status-im/nimbus-gui#13]
We should have a page that demos components and how they interact with each other so that one can see how to do things and what they end up looking like.

**Comment by GoNZooo** [comment]
@lazyfuhrer No, I'll leave it to you if you feel compelled to work on it. :)

**Comment by lazyfuhrer** [comment]
Sure, you can assign me @GoNZooo
